libcxx/utils/libcxx/test/params.py

```diff
@@ -330,5 +330,14 @@ def getModuleFlag(cfg, enable_modules):
             AddCompileFlag("-D_LIBCPP_REMOVE_TRANSITIVE_INCLUDES"),
         ],
     ),
+    Parameter(
```

Comment:
We could potentially remove `LIBCXX_EXECUTOR` if we have this setting? Then people could use `LIBCXX_TEST_PARAMS` to customize the executor.

It would be really nice to drop CMake options that are only related to testing, since we can pass them as Lit parameters.

If we do that, we'd want to add an entry to `releasenotes/18.rst` probably under Build System Changes.

Reply:
That sounds like a great idea. I'll submit a follow-up change for this.

Reply:
I think we should make it part of this change.